agent_build_refactored/managed_packages/scalyr_agent_libs/agent-libs-config

```diff
@@ -0,0 +1,75 @@
+#!/opt/scalyr-agent-2-dependencies/bin/python3
+# Copyright 2014-2022 Scalyr Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This script serves as executable that allows to create/recreate venv for the Linux Agent requirements.
+"""
+import argparse
+import pathlib as pl
+import shutil
+import subprocess
+
+DEPENDENCIES_PACKAGE_ROOT = pl.Path("/opt/scalyr-agent-2-dependencies")
+
+
+def initialize():
+    """
+    Initialize agent requirement libraries by creating virtual environment with them.
+    :return:
+    """
+    core_requirements_path = DEPENDENCIES_PACKAGE_ROOT / "core-requirements.txt"
+
+    # This is pre-created venv that is shipped with package.
+    venv_source_dir = DEPENDENCIES_PACKAGE_ROOT / "venv"
+
+    venv_dst_dir = pl.Path("/var/opt/scalyr-agent-2-dependencies/venv")
+    if venv_dst_dir.exists():
+        shutil.rmtree(venv_dst_dir)
+
+    # Copy venv that is shipped with the package.
+    shutil.copytree(
```

Comment:
If I understand it correctly, we utilize the same virtualenv for core agent dependencies + user dependencies?

Would it make sense to utilize different environment for those (e.g. to make updating easier or similar)?

Reply:
I've tried to make it clearer in this docstring

Reply:
I've tried to make it clearer in this docstring in this commit https://github.com/scalyr/scalyr-agent-2/blob/d48298a45a322219275048948038afdb0968664f/agent_build_refactored/managed_packages/managed_packages_builders.py#L73